solvers/PGD.py

```diff
@@ -34,6 +41,18 @@ def set_objective(self, A, reg, y, c, delta, data_fit):
         self.data_fit = data_fit
 
     def run(self, callback):
+        if self.prox_op == "condat_C":
+            prox_op = partial(ptv.tv1_1d, method='condat')
+        elif self.prox_op == "tv_mm":
+            prox_op = partial(tv_mm, max_iter=1000, tol=1e-6)
+        elif self.prox_op == "condat_numba":
+            prox_op = prox_condat
+        elif "gtv_mm" in self.prox_op:
+            K = int(self.prox_op[-1])
+            prox_op = partial(gtv_mm_tol2, max_iter=1000, tol=1e-6, K=K)
+        if self.prox_op != "condat_C":
+            jit_module()
```

Comment:
This should be called in `set_objective` and the `jit_module` should be called in the `warm_up` function that will be called only once (so you don't have to do it by yourself).